src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -211,6 +211,7 @@ public WorkspaceModel CurrentSpace
         /// <summary>
         /// Controls if the the ML data ingestion pipeline is beta from feature flag
         /// </summary>
+        [Obsolete("This flag is not needed anymore and will be removed in a future version of Dynamo. Use EnableDNADataIngestionPipeline instead")]
```

Comment:
I think we can remove this method. It was internal , I see no reason to keep it.